code/modules/antagonists/abductor/machinery/camera.dm

```diff
@@ -115,6 +115,11 @@
 	var/mob/living/carbon/human/C = owner
 	var/mob/camera/ai_eye/remote/remote_eye = C.remote_control
 	var/obj/machinery/abductor/pad/P = target
+
+	if(istype(get_area(remote_eye), /area/ai_monitored/turret_protected/ai))
+		to_chat(owner, "<span class='warning'>This area is too heavily shielded to safely transport to.</span>")
+		return
+
```

Comment:
based on what you said in the above comment, you're not interested in making this a flag?

Reply:
I mean sure, we can make it a flag, but are Abductors ever going to be further restricted in where they can teleport?
I'm totally fine with it being done but I don't think it matters at all.

Reply:
It's just cleaner code, and we don't know if we're going to have to apply it to something else in the future.